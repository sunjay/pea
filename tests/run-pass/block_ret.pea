fn add(a, b) {
    let sum = a + b;
    println!(sum);
    sum
}

fn main() {
    println!(add(1, 3));
    println!(add(add(33, -400), 19));
}
