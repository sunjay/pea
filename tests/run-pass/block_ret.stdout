4
4
-367
-348
-348
